CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# How to contribute to JuMP
+
+Welcome! This document explains some of the ways you can contribute to JuMP.
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the
+[JuMP Code of Conduct](https://github.com/JuliaOpt/JuMP.jl/blob/master/CODE_OF_CONDUCT.md).
+By participating, you are expected to uphold this code.
+
+## Join the Discourse forum
+
+First up, join the [Discourse forum](https://discourse.julialang.org/c/domain/opt).
+
+The forum is a good place to ask questions about how to use JuMP. You can also
+use the forum to discuss possible feature requests and bugs before raising a
+Github issue (more on this below).
+
+Aside from asking questions, the easiest way you can contribute to JuMP is to
+help answer questions on Discourse!
+
+## Improve the documentation
+
+Chances are, if you asked (or answered) a question on Discourse, then it is
+usually a sign that the [documentation](http://www.juliaopt.org/JuMP.jl/latest/)
+could be improved. Moreover, since it is your question, you are probably the
+best-placed person to improve it!
+
+The docs are written in Markdown and are built using [Documenter.jl](https://github.com/JuliaDocs/Documenter.jl).
+You can find the source of all the docs [here](https://github.com/JuliaOpt/JuMP.jl/tree/master/docs).
+
+If your change is small (like fixing typos, or one or two sentence corrections),
+the easiest way to do this is via Github's online editor. (Github has
+[help](https://help.github.com/articles/editing-files-in-another-user-s-repository/)
+on how to do this.)
+
+If your change is larger, or touches multiple files, you will need to make the
+change locally and then use Git to submit a pull request. (See [Contribute code to JuMP](#Contribute code to JuMP)
```

Comment:
This shows up in a funny way in the file, not sure what the fix is if any is needed.